egs2/aishell/asr1/README.md

```diff
@@ -1,3 +1,44 @@
+# Streaming Conformer + specaug + speed perturbation: feats=raw, n_fft=512, hop_length=128
```

Comment:
I'm curious if the real-time factor metric is also available for these models (probably for a specific GPU / CPU).